### PR TITLE
convert TestHelper.cpp with clang

### PR DESCRIPTION
**Pull request by winsvega**
this is what I got by executing clang-format 
```
YAML:36:23: error: unknown key 'SplitEmptyFunction'
  SplitEmptyFunction: false

```

### CODE REVIEW

**Comment by gumb0** [comment]
The result doesn't look right at all.
What's your clang-format version?

**Comment by chfast** [comment]
@winsvega You need to upgrade clang-format, because the version you have does not understand the config. If you tell me the version I can try to make it compatible (it is always backward compatible). But from the error I can tell you rather have to upgrade, because `SplitEmptyFunction` is a setting we really want to have.

**Comment by winsvega** [comment]
updated with clang-format 5

File: test/tools/libtesteth/TestHelper.cpp [hunk 1]
```diff
@@ -67,531 +66,540 @@ void connectClients(Client& c1, Client& c2)
...
-			using fsIterator = fs::directory_iterator;
-			for (fsIterator it(_dirPath); it != fsIterator(); ++it)
-			{
-				if (fs::is_regular_file(it->path()) && it->path().extension() == ext)
-					files.push_back(it->path());
-			}
-		}
-	}
-	return files;
+    for (auto& account : _o.count("alloc") ? _o["alloc"].get_obj() :
```
Comment (chfast) [inline test/tools/libtesteth/TestHelper.cpp]: This looks bad. We have 3 `:` here. Can you move the "accounts" initialization outside of the loop?

File: test/tools/libtesteth/TestHelper.cpp [hunk 2]
```diff
@@ -67,531 +66,540 @@ void connectClients(Client& c1, Client& c2)
...
 #else
-	boost::filesystem::path path(boost::filesystem::temp_directory_path() / boost::filesystem::unique_path());
-	string cmd = string("lllc ") + path.string();
-	writeFile(path.string(), _code);
-	string result = executeCmd(cmd);
-	boost::filesystem::remove(path);
-	result = "0x" + result;
-	checkHexHasEvenLength(result);
-	return result;
+    boost::filesystem::path path(
```
Comment (chfast) [inline test/tools/libtesteth/TestHelper.cpp]: Here you can save some space by `using fs =`.

**Comment by winsvega** [comment]
I have rebased the PR. looks like everything is at it's place.
but a quick review would be nice

**Pull request merged**